pkg/pc/fields/fields.go

```diff
@@ -58,6 +59,29 @@ type PodCluster struct {
 	Annotations Annotations
 }
 
+func (pc *PodCluster) Equals(other *PodCluster) bool {
+	if pc == nil && other == nil {
```

Comment:
You would still need the nil check to prevent a panic - I wanted to be explicit about the other properties I compared, but I'm not married to it